twine/settings.py

```diff
@@ -151,7 +153,13 @@ def verbose(self, verbose: bool) -> None:
         """Initialize a logger based on the --verbose option."""
         self._verbose = verbose
         root_logger = logging.getLogger("twine")
-        root_logger.addHandler(logging.StreamHandler(sys.stdout))
+        root_logger.addHandler(
+            rich.logging.RichHandler(
+                show_time=False,
+                show_path=False,
+                highlighter=rich.highlighter.NullHighlighter(),
+            )
```

Comment:
Without any of these options, the output would be:

<img width="672" alt="Screen Shot 2022-01-01 at 5 01 43 PM" src="https://user-images.githubusercontent.com/1326704/147861263-91fc31a4-c78d-47ca-8608-2f719712202f.png">